extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -64,27 +65,20 @@ export class BaseCarousel extends AMP.BaseElement {
 
 
   buildButtons() {
-    this.prevButton_ = this.element.ownerDocument.createElement('div');
+    // TODO(aghassemi erwinm, #4146) i18n
+    this.prevButton_ = addScreenReaderButton(this.element,
```

Comment:
Another options (which I am okay with) is carousel always adding extra screenreader buttons for next/previous and leaving the old buttons alone. Worse-case you endup with two next/previous buttons and that's not the end of the world.